src/com/houarizegai/calculator/Calculator.java

```diff
@@ -1,13 +1,35 @@
 
-package com.houarizegai.calculator;
+package Calculator.src.com.houarizegai.calculator;
```

Comment:
from `package com.houarizegai.calculator` to `package Calculator.src.com.houarizegai.calculator` 

This does not make any sense.